core/src/main/java/org/apache/gravitino/metalake/MetalakeManager.java

```diff
@@ -149,20 +185,23 @@ public BaseMetalake[] listMetalakes() {
    */
   @Override
   public BaseMetalake loadMetalake(NameIdentifier ident) throws NoSuchMetalakeException {
-    try {
-      BaseMetalake baseMetalake =
-          TreeLockUtils.doWithTreeLock(
-              ident,
-              LockType.READ,
-              () -> store.get(ident, EntityType.METALAKE, BaseMetalake.class));
-      return newMetalakeWithResolvedProperties(baseMetalake);
-    } catch (NoSuchEntityException e) {
-      LOG.warn("Metalake {} does not exist", ident, e);
-      throw new NoSuchMetalakeException(METALAKE_DOES_NOT_EXIST_MSG, ident);
-    } catch (IOException ioe) {
-      LOG.error("Loading Metalake {} failed due to storage issues", ident, ioe);
-      throw new RuntimeException(ioe);
-    }
+    return TreeLockUtils.doWithTreeLock(
+        ident,
+        LockType.READ,
+        () -> {
+          try {
+            BaseMetalake baseMetalake = store.get(ident, EntityType.METALAKE, BaseMetalake.class);
```

Comment:
Why do we still need to get the metalake from store, rather than from cache? With your code, what's the meaning of `METALAKE_CACHE.put(ident, baseMetalake)`?

Reply:
It's a mistake.